### ISSUE TITLE
error at log, and integration not working at all

### ISSUE THREAD

**Issue by KitHubek** [viktak/ha-cc-abalin-nameday#3]
Logger: homeassistant.components.sensor
Source: custom_components/abalin_nameday/sensor.py:62
Integration: Sensor ([documentation](https://www.home-assistant.io/integrations/sensor), [issues](https://github.com/home-assistant/home-assistant/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+sensor%22))
First occurred: 19:28:42 (1 occurrences)
Last logged: 19:28:42

abalin_nameday: Error on device update!
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 493, in _async_add_entity
    await entity.async_device_update(warning=False)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 715, in async_device_update
    await task
  File "/usr/local/lib/python3.10/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/abalin_nameday/sensor.py", line 98, in update
    self.data.update(self.type)
  File "/usr/src/homeassistant/homeassistant/util/__init__.py", line 192, in wrapper
    result = method(*args, **kwargs)
  File "/config/custom_components/abalin_nameday/sensor.py", line 62, in update
    self.data = json.loads(myNameDay.GetData())
  File "/usr/local/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python3.10/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 42 (char 41)

**Comment by viktak** [comment]
I also had the same experience yesterday, but today it seems to be working for me - I made no changes to this add-on, and the upstream API has not changed either (just spoke to the developer/maintainer), so it seems to be some temporary issue that is gone now.

I just tested it on 2022.11.4, and it works. If it is still not working for you try restarting HA, maybe the whole box. If the issue still persists, give me more details on the environment you are using, etc.
